Comments/ClientApp/src/components/LeadInfo/LeadInfo.js

```diff
@@ -0,0 +1,60 @@
+﻿import React, { Component } from "react";
+import { withRouter } from "react-router-dom";
+import Cookies from "js-cookie";
+import { withHistory } from "../HistoryContext/HistoryContext";
+import BreadCrumbsWithRouter, { Breadcrumbs } from "../Breadcrumbs/Breadcrumbs";
+import { load } from "../../data/loader";
+import { queryStringToObject } from "../../helpers/utils";
+
+export const LeadInfo = () => {
+	const emailAddress = "DITApplicationSupport@nice.org.uk"
+	const emailSubject = encodeURIComponent("New commenting lead request");
+	const emailBody = encodeURIComponent("Dear Sir/Madam\r\rMy name:\rI want to be commenting lead for\r\rKind regards");
+
+	return (
+		<div className="container">
+
+			<Breadcrumbs links={[{label:"Home",url:"/"},{label:"All consultations", url:"/guidance/inconsulation"}]} />
+
+			<div className="grid">
+				<div data-g="12">
```

Comment:
```suggestion
		<div className="container">
                        <div className="grid">
			        <div data-g="12">
				         <Breadcrumbs links={[{label:"Home",url:"/"},{label:"All consultations", url:"/guidance/inconsulation"}]} />
```

Sorry, I advised you wrong on this I think... The breadcrumb should be in a grid-12 div.

Reply:
PS. The tabs and formatting are off on my suggestion